extensions/Push/lib/Connector/Webhook.pm

```diff
@@ -20,6 +20,7 @@ use Bugzilla::Attachment;
 use Bugzilla::Extension::Webhooks::Webhook;
 use Bugzilla::Extension::Push::Constants;
 use Bugzilla::Extension::Push::Util;
+use Bugzilla::Extension::Push::Serialize qw(_integer _boolean _bug);
```

Comment:
Sorry for the delay with this review. The company gave us all a few days off for the holiday in the US. 

This will not work unfortunately the way you have it as Serialize.pm does not export those functions using Exporter.pm. If I make a change to a private but I get a function not found error when the webhook is processed.

An example of using Exporter.pm can be seen in https://github.com/mozilla-bteam/bmo/blob/master/extensions/Push/lib/Util.pm#L23-L35

The easiest fix would be to copy the _int() and _boolean() functions from Serialize.pm and put them in the Webhook.pm module since they are very small.

As for the call to _bug(), this is not necessary as the data from `$message->payload_decoded` should be ok to use directly.

Reply:
Oh yes, sorry about that.